.travis.yml

```diff
@@ -37,6 +37,128 @@ submit_to_coverity_scan: &submit_to_coverity_scan
                "https://scan.coverity.com/builds";
   fi
 
+build_simulator: &build_simulator
+  - cd "${TRAVIS_BUILD_DIR}/ports/freertos-posix"
+  - git clone -b "v9.0.0" --single-branch https://github.com/shlinym/FreeRTOS-Sim.git FreeRTOS-Sim
```

Comment:
The example I wrote just happens to use this specific fork of FreeRTOS-Sim, but I think simply depending on third-party repository existence isn't wise. A better idea would be to clone the repository under an ADLINK owned GitHub repository? That way we can somewhat control what goes in etc.